shortcodes.php

```diff
@@ -380,56 +380,58 @@ function sc_org_chart($attrs) {
 		'orderby'     => 'menu_order',
 		'order'       => 'ASC',
 	)) : false;
-	
-	function display_people($category){
-		$people = get_posts(array(
-			'numberposts' => -1,
-			'post_type'   => 'profile',
-			'category'    => $category->term_id,
-			'orderby'     => 'menu_order',
-			'order'       => 'ASC',
-		));
 
-		?>
-		<div class="row">
-		<?
-		$count = 0;
-		foreach($people as $person) {
-			if($count > 0 && ($count % 4) == 0) {
-				echo '</div><div class="row">';
-			}
-			$department_text = get_post_meta($person->ID, 'profile_department_text', True);
-			$department_url  = get_post_meta($person->ID, 'profile_department_url', True);
-			$email           = get_post_meta($person->ID, 'profile_email', True);
+	if ( !function_exists( 'display_people' ) ) {
```

Comment:
I think that error was showing up because that chunk of code was getting called multiple times, redeclaring that function each time. I moved it out to functions.php (a better place for it) and was able to remove the function_exists().